portfolio/src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.data.Location;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns Location data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/location-data")
+public class LocationServlet extends HttpServlet {
+
+  private Collection<Location> israelLocations;
+
+  @Override
+  public void init() {
+    israelLocations = new ArrayList<>();
+
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/location-data.csv"));
+    while (scanner.hasNextLine()) {
+        String line = scanner.nextLine();
+        String[] cells = line.split("\\s*,\\s*");
+        if (cells.length == 2) {
+            try {
+                double lat = Double.parseDouble(cells[0]);
+                double lng = Double.parseDouble(cells[1]);
+                israelLocations.add(new Location(lat, lng));
+            } catch (NumberFormatException e) {
+              continue;
```

Comment:
Don'y you want to do something here? Propagate the exception / log? Is this scenario applicable and may be ignored?

Reply:
I didn't have an idea for something better to do than just skip this location and not try to show it on map, since it's not an input from the user I didn't want the user to get any errors or instructions. 
maybe it can be useful as log thow so I will add it